### PR TITLE
Feat/new topic filter feature

### PR DESCRIPTION
**Pull request by vvandijck**
This pull-request includes:
- New `Topics` component (will eventually replace the `Filter` component)
- Updated the `/team` and `/updates` pages to use the new `Topics` component and filtering
functionality.

### CODE REVIEW

**Comment by JesseE** [comment]
My advice to you is to create smaller pull requests. Its quite the chore to review this pull request. It would be easier to accurately review this pull request if it was split in smaller pull request. It would provide a better overview of changes.

File: components/team-members-overview/team-members-overview.js [hunk 1]
```diff
@@ -9,170 +9,90 @@ class TeamMembersOverview extends Component {
...
+            topics={departments}
+            onTopicChanged={this.changeTopicHandler} />
         </div>
 
         {introText && <p className="team-members-intro-text container">{introText}</p>}
         <ul className="team-members-overview container">
-          {filteredTeam.map((teamMember, index) => (
-            <TeamMember key={index} data={teamMember} />
+          {filteredTeam.map((member, index) => (
+            <TeamMember key={index} data={member} />
```
Comment (JesseE) [inline components/team-members-overview/team-members-overview.js]: you could write this as a one liner without the index parameter and use the name of the member as key:
 {filteredTeam.map(member => <TeamMember key={member.name} data={member} />)}

File: components/update-overview/update-overview.js [hunk 2]
```diff
@@ -1,57 +1,104 @@
...
+      .filter(update => this.hasSelectedTopic(update, topic))
+  }
+
+  isHighlightedUpdate(update) {
+    const { data } = this.props
+    return !data.highlights.some(highlight => highlight.id === update.id)
+  }
+
+  hasSelectedTopic(update, topic) {
+    if (topic === 'All')
```
Comment (JesseE) [inline components/update-overview/update-overview.js]: a nit pick but you could write this shorter like:
if(topic === 'All') return true
return update.category.name === topic

File: components/update-overview/update-overview.less [hunk 3]
```diff
@@ -4,53 +4,76 @@
 .update-overview {
   .clearfix();
 
-  .update-extract-small {
-    max-width: 450px;
-    margin-right: auto;
-    margin-left: auto;
+  > .container-inner {
```
Comment (JesseE) [inline components/update-overview/update-overview.less]: this looks weird to me. not sure if this works or is needed. isnt it the save if you would remove the >

Comment (vvandijck) [reply components/update-overview/update-overview.less]: We need this styling only on the immediate child `.container-inner`, not on any other `.container-inner`.

Comment (JesseE) [reply components/update-overview/update-overview.less]: Ah yes I understand

**Pull request merged**